### ISSUE TITLE
Test suite: Tracking issue

### ISSUE THREAD

**Issue by Narrat** [dyne/tomb#535]
Some points to possible enhance the tomb testsuite

- [ ] Update sharness (https://github.com/felipec/sharness/releases)
- [ ] bind test pollutes `$HOME`
  * remove generated file?
  * set up a temp `$HOME` in general?
 - [ ]  restructure tests
   * sorting after tomb features is only partially helpful, as there may be testcases that won't really fit anywhere then
 -[] General: cover and test more

and likely more. Those will appear when this is tackled

**Comment by jaromil** [comment]
it may be worth looking into a switch to the bats test framework which we use extensively in zenroom so there is a bit more in house Knowledge about that in dyne.

**Comment by Narrat** [comment]
That could indeed be worthwhile to use what is already known. Is it a general shell test framework? The Readme itself explicitly states bash: `Bats is a [TAP](https://testanything.org/)-compliant testing framework for Bash 3.2 or above.`

**Comment by roddhjav** [comment]
> it may be worth looking into a switch to the bats test framework

I agree. I wanted to give it a try a while ago. however, I have never had time to spend on it.
